css/mystyle.css

```diff
@@ -68,7 +67,7 @@ ol, ul {
 	box-shadow: 0px 0px 20px 0px #000;
 }
 .top-navigation > .content {
-	width: 970px;
+	max-width: 970px;
```

Comment:
Setting a width won't allow the page to be responsive. Changed it to "max-width" instead so the page will get smaller on smaller screens.